csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "supports": {},
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "NuSpec.ReferenceGenerator": { "type": "build", "version": "2.0.0-beta-bld14" }
+  },
+  "frameworks": {
+    "netstandard1.0": {}
```

Comment:
I THINK you can't have just netstandard1.0, you also need profile259 here, because for if someone is trying to use Google.Protobuf from VS2013, he'll have an old version of nuget.exe and that one knows nothing about netstandard1.0